modules/components/themes/_bootstrap3-theme.scss

```diff
@@ -0,0 +1,38 @@
+.ng2-tag-input.bootstrap3 {
```

Comment:
Hi @Vrael,

thanks for this addition.

Would it be possible to format this file better?
ex. indentation, spaces between selectors

Thanks!